pkg/lib/cockpit-components-listing.jsx

```diff
@@ -253,21 +255,30 @@ export class ListingRow extends React.Component {
                 );
             }
 
+            let simpleBody, heading;
+            if ('simpleBody' in this.props) {
```

Comment:
I think we usually do `if (this.props.simpleBody)`, not that it matters much :)

Reply:
It's more because the exact same formulation is used for listingDetail just above it.